project.clj

```diff
@@ -28,27 +30,34 @@
                          [net.logstash.logback/logstash-logback-encoder "4.7"]
                          [org.codehaus.janino/janino "2.7.8"]
                          [com.fasterxml.jackson.core/jackson-core "2.7.3"]
+                         [org.yaml/snakeyaml "1.13"]
```

Comment:
Both clj-yaml and JRuby depend upon snakeyaml.  The snakeyaml version that clj-yaml 0.4.0 depends upon (which I think is still the latest one) is older than the snakeyaml JRuby uses - 1.5.  I think we may have gone with "1.13" because that's what JRuby was using - although I can't remember if we played with / ran into problems trying to use the older one from Puppet Server.

Since we have clj-yaml in here already, I think it makes sense to include snakeyaml as well so that we ensure that the newer version is pulled in by default.  That said, I don't know what all projects besides Puppet Server use clj-yaml even though it seems like it could be a reasonably common thing.